### PR TITLE
Marathon Api port name (#370)

### PR DESCRIPTION
**Pull request by ignacioPL**
-Use the port name from the lookup if present, otherwise use it from the config

### CODE REVIEW

File: discovery-marathon-api/src/main/scala/akka/discovery/marathon/MarathonApiSimpleServiceDiscovery.scala [hunk 1]
```diff
@@ -84,6 +84,11 @@ class MarathonApiSimpleServiceDiscovery(system: ActorSystem) extends SimpleServi
 
     log.info("Requesting seed nodes by: {}", request.uri)
 
+    val portName = lookup.portName match {
```
Comment (raboof) [inline discovery-marathon-api/src/main/scala/akka/discovery/marathon/MarathonApiSimpleServiceDiscovery.scala]: Is this variable used anywhere?

Comment (ignacioPL) [reply discovery-marathon-api/src/main/scala/akka/discovery/marathon/MarathonApiSimpleServiceDiscovery.scala]: My bad, I commit the change with the use in targets

Comment (raboof) [reply discovery-marathon-api/src/main/scala/akka/discovery/marathon/MarathonApiSimpleServiceDiscovery.scala]: Aah, that makes a little more sense. Shouldn't it also be used while constructing the query somehow, though?

Comment (ignacio-dc) [reply discovery-marathon-api/src/main/scala/akka/discovery/marathon/MarathonApiSimpleServiceDiscovery.scala]: you can't filter by portname in the marathon Rest API

Comment (ignacioPL) [reply discovery-marathon-api/src/main/scala/akka/discovery/marathon/MarathonApiSimpleServiceDiscovery.scala]: you can't filter by portname in the marathon Rest API

**Comment by chbatey** [comment]
Thanks @ignacioPL

**Pull request merged**